src/Controller/AgendaController.php

```diff
@@ -186,20 +188,36 @@ public function create(MunicipalityHelper $municipalityHelper, Request $request)
     }
 
     /**
-     * @Route("/{id}", name="agenda_delete", methods={"DELETE"})
+     * @Route("/{id}", name="agenda_delete", methods={"POST"})
```

Comment:
`methods` should allow both `GET` (form rendering the form) and `DELETE` for actually deleting the agenda.